scripts/roles.json

```diff
@@ -3961,7 +4354,7 @@
     "requiredContextRole": ["group", "menu", "menubar"],
     "requiredOwnedElements": [],
     "requiredProps": ["aria-checked"],
-    "superClass": ["checkbox", "menuitem"]
```

Comment:
This is still superclassed under `checkbox`:

https://www.w3.org/TR/wai-aria-1.2/#menuitemcheckbox

Reply:
Not in the latest editor's draft.